src/main/java/core/basesyntax/service/impl/BalanceOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import static javax.swing.UIManager.put;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {

```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
please fix the same issue with all handlers